packages/wonder-blocks-testing/src/fixtures/__tests__/setup.test.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import {isolateModules} from "../../jest/isolate-modules.js";
+
+describe("#getConfiguration", () => {
+    it("should return the configuration passed during setup", () => {
+        // Arrange
+        const {setup, getConfiguration} = isolateModules(() =>
+            require("../setup.js"),
+        );
+        const configuration = {
+            adapter: {
+                name: "mytestadapter",
+                declareGroup: jest.fn(),
+            },
+            defaultAdapterOptions: {},
+        };
+        setup(configuration);
+
+        // Act
+        const result = getConfiguration();
+
+        // Assert
+        expect(result).toBe(configuration);
+    });
+
+    it("should throw if setup has not been performed", () => {
+        // Arrange
+        const {getConfiguration} = isolateModules(() => require("../setup.js"));
+
+        // Act
+        const act = () => getConfiguration();
+
+        // Assert
+        expect(act).toThrowErrorMatchingInlineSnapshot(`"Not configured"`);
+    });
+});
+
+describe("#setup", () => {
+    it("should set the configuration returned by getConfiguration", () => {
```

Comment:
The test case title, for me, represents the requirement, not the implementation. It is a requirement that it set the config that getConfiguration returns, but not necessarily a requirement that it can be called more than once.